src/vector/backends/numpy.py

```diff
@@ -1311,7 +1332,7 @@ def __setitem__(self, where: typing.Any, what: typing.Any) -> None:
         return _setitem(self, where, what, False)
 
 
-class MomentumNumpy2D(PlanarMomentum, VectorNumpy2D):
+class MomentumNumpy2D(PlanarMomentum, VectorNumpy2D):  # type: ignore[misc]
```

Comment:
The `type: ignore[misc]` comments were already present in `VectorNumpy*D` definitions, but now they are also required for `Momentum` classes.